pages/windows/psexec.md

```diff
@@ -0,0 +1,25 @@
+# PsExec
```

Comment:
```suggestion
# psexec
```

Reply:
> ```suggestion
> # psexec
> ```

I have a doubt recently I came to know there wouldn't be any linter errors for using Capital characters in heading. I even check this file in `tldrl`. Is this the cause of error here?